README.md

```diff
@@ -48,28 +48,28 @@ For each CODATA release, we provide two maps; one for the data values and one fo
-| Stefan-Boltzmann constant         | `stefanBoltzmann`     | 5.670 367 e-8         | 0.000 013 e-8       | W m^-2 K^-4    | 
+  | Quantity                          | Variable name         | Value                 | Uncertainty         | Unit           | 
+  | --------                          | -------------         | ----                  | ------              | ------         | 
+  | Avogadro constant                 | `avogadro`            | 6.022 140 76 e23      | 0.0                 | mol^-1         | 
+  | Boltzmann constant                | `boltzmann`           | 1.380 649 e-23        | 0.0                 | J K^-1         | 
+  | Boltzmann constant                | `k`                   | 1.380 649 e-23        | 0.0                 | J K^-1         | 
+  | conductance quantum               | `conductanceQuantum`  | 7.748 091 729 e-5     | 0.0                 | S              | 
+  | conductance quantum               | `G0`                  | 7.748 091 729 e-5     | 0.0                 | S              | 
+  | Elementary charge                 | `elementaryCharge`    | 1.602 176 6234 e-19   | 0.0                 | C              | 
+  | classical electron radius         | `electronRadius`      | 2.817 940 3227 e-15   | 0.000 000 0019 e-15 | m              | 
+  | Faraday constant                  | `faraday`             | 96 485.332 89         | 0.000 59            | C mol^-1       | 
+  | fine-structure constant           | `fineStructure`       | 7.297 352 5664 e-3    | 0.000 000 0017 e-3  | --             | 
+  | magnetic constant                 | `magnetic`            | 12.566 370 614... e-7 | 0.0                 | N A^-2         | 
+  | magnetic flux quantum             | `magneticFluxQuantum` | 2.067 833 831 e-15    | 0.000 000 013 e-15  | Wb             | 
+  | molar gas constant                | `molarGas`            | 8.314 4598            | 0.000 0048          | J mol^-1 K^-1  | 
+  | Newtonian constant of gravitation | `G`                   | 6.674 08 e-11         | 0.000 31 e-11       | m^3 kg^-1 s^-2 | 
+  | Newtonian constant of gravitation | `newtonianGravity`    | 6.674 08 e-11         | 0.000 31 e-11       | m^3 kg^-1 s^-2 | 
+  | Planck constant                   | `planck`              | 6.626 070 15 e-34     | 0.0                 | J s            | 
+  | Planck constant                   | `h`                   | 6.626 070 15 e-34     | 0.0                 | J s            | 
+  | Rydberg constant                  | `rydberg`             | 10 973 731.568 160    | 0.000 021           | m^-1           | 
+  | Speed of light in a vacuum        | `c`                   | 299 792 458           | 0.0                 | m s^-1         | 
+  | Speed of light in a vacuum        | `speedOfLight`        | 299 792 458           | 0.0                 | m s^-1         | 
+  | Stefan-Boltzmann constant         | `stefanBoltzmann`     | 5.670 374 419 e-8     | 0.0                 | W m^-2 K^-4    | 
```

Comment:
Why did some of these values and/or uncertainties change (not talking about the once where you remove the scientific notation)?

Reply:
They changed because I had them in wrong. There are two tables; one for CODATA2014 and one for CODATA2018. When I was updating the values, I copied/pasted the table and the updated the values. At some point I started changing the wrong table. This is the fix for that.

Note that these changes are only in the README file.